backend/ops_api/ops/utils/auth_views.py

```diff
@@ -108,6 +114,9 @@ def _get_token_and_user_data_from_internal_auth(user_data: dict[str, str]):
         # authZ, given a valid login from the prior AuthN steps above.
 
         additional_claims = {}
+        additional_claims["iss"] = "https://opre-ops-backend"  # Placeholder
```

Comment:
not sure of the function of the placeholder but the string is duplicated - can it be put in a constant instead?  Does the value, e.g. `https://opre-ops-backend` have any meaning or can empty string or None be used instead?

Reply:
Basically, anything is allowed; it simply represents principal that issued the JWT, so that you can validate it wherever needed (so ensure in the frontend, that a specific entity generated the JWT. The placeholder is currently used and checked; Some value is preferred in a default (enabled) check state, versus an empty string (tyranny of defaults). It probably should be in the config or a constant; but wasn't sure "what" to use yet 🤔

Reply:
moved these to the config via: `JWT_ENCODE_ISSUER` and `JWT_ENCODE_AUDIANCE`